### ISSUE TITLE
Hint item award in quest with specific suffix or prefix.

### ISSUE THREAD

**Issue by smoothGrace** [DinoDevs/GladiatusCrazyAddon#162]
it use in under world,some times,the quest award is contain some good suffix or prefix to smeltery,for the underworld recipes.

change background-color to hint?

the better to add a list container in setting-pantheon,add which the user want.default blank.

**Comment by GramThanos** [comment]
The problem with this is the translations of the prefixes and suffixes. We can auto load what ever a player has in his forge lists, but these will probably be the ones that he doesn't want to gather any more.

**Comment by smoothGrace** [comment]
how about simulate items of prefixes and suffixes to get local translations type?
once simulates every prefixes and suffixes and record them into config file,for later using?
or allow users to upload all prefixes and suffixes on item or even  in local language to your data center,fixing when appear,keep in next version's config file?default with english?

**Comment by smoothGrace** [comment]
and,a filter to defaul hide prefixes and suffixes that already learned,
and,not tip all food.

**Comment by Tharacius** [comment]
I would also love a feature like this.

**Comment by GramThanos** [comment]
Indeed, having the lists will allow us to implement other features too.

**Comment by FrutyX** [comment]
Related to #298.

For now, maybe @GreatApo can make the quest list show the names of the item without hovering on it (related to #265).

No further progress on this is possible.

**Comment by FrutyX** [comment]
I wonder if we really need this now, me and Apo managed to show the item names without needing to hover over the small item image, so with your own eyes and a quick look, you can see every item.

**Comment by GreatApo** [comment]
> I wonder if we really need this now, me and Apo managed to show the item names without needing to hover over the small item image, so with your own eyes and a quick look, you can see every item.

Probably will never be scripted but leave it since we have/had a feature that highlights the quests relative tot he area you last attached

**Comment by smoothGrace** [comment]
It use to find item in underground quest with underground only prefix of
suffix.

When a micro event with -50% quest cool is hold,and Use Secret Knowledge
-50%quest cool,it will be 0 cooldown for reset quest,when character is in
underground,it can reset with no limits,and get item who need,save for the
smelt event to get 10% or 20% chance to get the rare scroll.

Apostolos Grammatopoulos ***@***.***> 於 2022年11月14日 週一
凌晨4:34寫道：
